sycl/source/detail/ur_device_info_ret_types.inc

```diff
@@ -129,6 +129,7 @@ MAP(UR_DEVICE_INFO_ATOMIC_MEMORY_ORDER_CAPABILITIES, ur_memory_order_capability_
 MAP(UR_DEVICE_INFO_ATOMIC_MEMORY_SCOPE_CAPABILITIES, ur_memory_scope_capability_flags_t)
 MAP(UR_DEVICE_INFO_ATOMIC_FENCE_ORDER_CAPABILITIES, ur_memory_order_capability_flags_t)
 MAP(UR_DEVICE_INFO_ATOMIC_FENCE_SCOPE_CAPABILITIES, ur_memory_scope_capability_flags_t)
+MAP(UR_DEVICE_INFO_LUID, std::array<uint8_t, 8>)
```

Comment:
Was the script in the comments above able to add it, or did you add it manually? If the script can't, then maybe we need to move this line further down.